orttraining/orttraining/test/python/checkpoint/orttraining_test_backend_api.py

```diff
@@ -0,0 +1,106 @@
+import os
```

Comment:
It would appear that these tests are not being called. Since these are distributed tests, you will need to call them from the ```orttraining_test_checkpoint.py``` file. Let me know if you need help with that.

Reply:
Added to `orttraining_test_checkpoint.py`